### DISCUSSION TITLE
solokey2 not recognised in chromium based browsers

### DISCUSSION THREAD

**Discussion by papezg** [solokeys/solo2#145] [Q&A]
I have solokey2 with firmware 2:20220822.0 and since latest update of the firmware all chrome based browsers doesn't recognise the key. Is there any hidden flag to enable it? To be more specific it doesn't work in Wordpress 2FA authentication solution. The test on demo.yubikey.com is working OK.

**Comment by nickray** [comment]
Is there a Wordpress demo we could test? Also is that definitely FIDO, not Yubico OTP or similar?

**Comment by theerror** [comment]
Actually, I've just hit the same now with Google in Brave & Vivaldi. It starts to wink, response to touch, but nothing is happening and dialog for confirmation still remains. However it does work for example here: 
https://www.token2.com/tools/fido2-test/ or https://webauthn.me/debugger

However - on google page, it blinks the stragne way, that there are blue and green colours together and by touching, only blue remains. When I do test on the test, page, where it works, it's just blue blinking changed to green and login.

**Reply by cflee** [reply]
+1, I just ran into this when registering on both GitHub.com and Fastmail.com. It alternates between blue and green, and it only works when I time my tap to hit on blue.
